build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,332 @@
+apiVersion: v1
+data:
+  Install-WindowsCNI-Containerd.ps1: |
+    $ErrorActionPreference = "Stop";    
+    mkdir -force c:/var/log/antrea
+
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/') 
+    mkdir -force C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/ca.crt C:/var/run/secrets/kubernetes.io/serviceaccount
+    cp $mountPath/var/run/secrets/kubernetes.io/serviceaccount/token C:/var/run/secrets/kubernetes.io/serviceaccount
+    mkdir -force c:/opt/cni/bin/
+    mkdir -force c:/etc/cni/net.d/
+    cp $mountPath/k/antrea/cni/* c:/opt/cni/bin/
+    cp $mountPath/etc/antrea/antrea-cni.conflist c:/etc/cni/net.d/10-antrea.conflist
+    mkdir -force c:/k/antrea/bin
+    cp $mountPath/k/antrea/bin/antctl.exe c:/k/antrea/bin/antctl.exe
+    
+  Run-AntreaAgent-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
+    $mountPath =  ($mountPath.Replace('\', '/')).TrimEnd('/')
+    & "$mountPath/k/antrea/bin/antrea-agent.exe" --config=$mountPath/etc/antrea/antrea-agent.conf --logtostderr=false --log_dir=c:/var/log/antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4 --v=0
+
+  Run-AntreaOVS-Containerd.ps1: |
+    $ErrorActionPreference = "Stop"
+    $OVSInstallDir = "C:\openvswitch"
+    $OVS_DB_SCHEMA_PATH = "$OVSInstallDir\usr\share\openvswitch\vswitch.ovsschema"
+    $OVS_DB_PATH = "$OVSInstallDir\etc\openvswitch\conf.db"
+    if ($(Test-Path $OVS_DB_SCHEMA_PATH) -and !$(Test-Path $OVS_DB_PATH)) {
+        Set-Location -Path "C:\openvswitch\usr\bin
+        ovsdb-tool create "$OVS_DB_PATH" "$OVS_DB_SCHEMA_PATH"
+    }
+    Set-Location -Path "C:\openvswitch\usr\sbin
+    ovsdb-server $OVS_DB_PATH -vfile:info --remote=punix:db.sock --remote=ptcp:6640 --log-file=/var/log/antrea/openvswitch/ovsdb-server.log --pidfile --detach
+    Set-Location -Path "C:\openvswitch\usr\bin"
+    ovs-vsctl --no-wait init
+
+    # Set OVS version.
+    $OVS_VERSION=$(Get-Item $OVSInstallDir\driver\OVSExt.sys).VersionInfo.ProductVersion
+    ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
+
+    Set-Location -Path "C:\openvswitch\usr\sbin"
+    ovs-vswitchd --log-file=/var/log/antrea/openvswitch/ovs-vswitchd.log --pidfile -vfile:info --detach
+
+    while ($true) {
+      if ( !( Get-Process ovsdb-server ) ) {
+        Set-Location -Path "C:\openvswitch\usr\sbin"
+        ovsdb-server $OVS_DB_PATH -vfile:info --remote=punix:db.sock --remote=ptcp:6640 --log-file=/var/log/antrea/openvswitch/ovsdb-server.log --pidfile --detach
+      }
+      if ( !( Get-Process ovs-vswitchd ) ) {
+        Set-Location -Path "C:\openvswitch\usr\sbin"
+        ovs-vswitchd --log-file=/var/log/antrea/openvswitch/ovs-vswitchd.log --pidfile -vfile:info --detach
+      }
+      Start-Sleep -Seconds 30
```

Comment:
Use a sleep interval variable: Instead of hardcoding the sleep duration, you can use a variable to store the duration. This makes it easier to modify in the future if needed.